src/Slackbot.Net.Extensions.FplBot/Handlers/FplTransfersCommandHandler.cs

```diff
@@ -48,39 +52,48 @@ public async Task<HandleResponse> Handle(SlackMessage message)
             var players = await playersTask;
             var gw = await gameweekTask;
 
-            var transferTasks = league.Standings.Entries
-                .Select(entry => _transfersClient.GetTransfers(entry.Entry))
-                .ToArray();
+            var entryTasks = league.Standings.Entries
+                .ToDictionary(entry => entry.Entry, entry => new
+                {
+                    TransferTask = _transfersClient.GetTransfers(entry.Entry),
+                    PickTask = _entryClient.GetPicks(entry.Entry, gw.Value)
+                });
 
             var entryTransfers = new List<EntryTransfer>();
```

Comment:
Kanskje det her kunne vært en funksjon som tok inn `Entry` i stedet for, som du kan kalle fra den neste loopen din? For nå itererer du `league.Standings.Entries` tre ganger hheheh

Reply:
Enig, godt poeng 👍

Reply:
Refaktorert